netbox/ipam/lookups.py

```diff
@@ -177,6 +177,11 @@ class Host(Transform):
     lookup_name = 'host'
 
 
+class HostAsInet(Transform):
```

Comment:
This feels redundant to the existing `Host` and `Inet` transforms. Is there a way we can utilize those instead of introducing a new transform? We could register these transforms for IPAddressField:

```python
IPAddressField.register_lookup(lookups.Host)
IPAddressField.register_lookup(lookups.Inet)
```

This allows us to chain lookups:

```
>>> IPRange.objects.filter(start_address__host__inet__lte='192.168.1.50')
<RestrictedQuerySet [<IPRange: 192.168.1.0-49/22>, <IPRange: 192.168.1.50-99/22>]>
```

Reply:
Agreed, this does seem to work fine (unit tests pass).